### ISSUE TITLE
Feature: Basic Contact Details

### ISSUE THREAD

**Issue by Tethik** [polyhere/meta#3]
**Current Progress ⌛**
I have a basic MVP webapp set up, main focus has been on getting a basis for a mobile focused app with offline storage. 

Tasks to make this a reality:
- [x] Webapp with offline storage
- [ ] Server sync for offline storage - so you can have multiple synced clients and don't lose your data.
- [ ] Clientside Encryption - so that we as admins cannot read the sensitive data.

**Functionality for the User**

Ideas for data you may want to store about a person (likely everything being optional):
- Name or nickname
- Address
- Contact details (email, phone)
- Birthday and ideas for gifts
- Type of relationship with that person
- Anniversary date

... or any arbitrary free-form data which the user can pick (but the above give some hints toward how different premade data structures can look). 

Feel free to comment or add your own wants here :)

**Open questions 🤔**
Need a better name than "Contacts". Looking for something warmer.

**Comment by AKAmasha** [comment]
does it have to be one word? and is it only for the name in the category, so it will only show up as "contacts" or also "contact" etc?
tbh contacts is pretty intuitive and the minimalism and ease of use may suffer from fancy new words for functional things

**Comment by Tethik** [comment]
> does it have to be one word? and is it only for the name in the category, so it will only show up as "contacts" or also "contact" etc? tbh contacts is pretty intuitive and the minimalism and ease of use may suffer from fancy new words for functional things

No, it does not have to be one word. We can put whatever label we want on it and have the correct plural/non-plural word.  Technically, we could even let the user themselves define the words for it that are display in their UI (I like the idea of making the app a bit customizable and more personal to the user).

True, it's fairly well known and intuitive. I just wish for a warmer word to communicate that these are people close or intimate or dear to me (friends, family, lovers).
